scripts/minikube-rbac.yaml

```diff
@@ -0,0 +1,78 @@
+# Wide open access to the cluster (mostly for kubelet)
```

Comment:
To be honest, i am not familiar with rbac. But i think @smarterclayton could take a look at this.

Reply:
the `scripts/minikube-rbac.yaml` is just some rbac roles to give to kube-dns, etc. as minikube doesn't provision them (RE: https://github.com/kubernetes/minikube/issues/1734#issuecomment-323443032), probably a good idea to put the link into the file

Reply:
Agree. Add the link or the issue to the comment is quite useful. I am not getting the meaning of `sd` when first review this. :)